src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,7 @@
   <data name="ERR_NameNotInContext" xml:space="preserve">
     <value>The name '{0}' does not exist in the current context.</value>
   </data>
+  <data name="HideBase" xml:space="preserve">
+    <value>Hide base member</value>
```

Comment:
Hmmm... "Hide" or "Shadow"? I think I'm OK with either, so Hide is fine with me. What do you think?